pkg/cli/cmds/server.go

```diff
@@ -208,20 +209,25 @@ var ServerFlags = []cli.Flag{
 	ClusterDomain,
 	cli.StringFlag{
 		Name:        "flannel-backend",
-		Usage:       "(networking) backend<=option1=val1,option2=val2> where backend is one of 'none', 'vxlan', 'ipsec', 'host-gw', 'wireguard-native', or 'wireguard' (deprecated)",
+		Usage:       "(deprecated) use --flannel-opts",
```

Comment:
I wonder if we should add `--disable-flannel` or something?

Hiding the option to disable flannel even further, when it's a very common option for users that want to deploy their own CNI, seems like a step backwards from a UX perspective.

Reply:
IIRC, `disable-flannel` was deprecated and removed